src/core/lib/iomgr/tcp_server_uv.c

```diff
@@ -76,13 +76,30 @@ struct grpc_tcp_server {
 
   /* shutdown callback */
   grpc_closure *shutdown_complete;
+
```

Comment:
How about "Pool of resources available to the libuv-based TCP server" (if that last bit is accurate).